src/translations/translationHelper.js

```diff
@@ -0,0 +1,49 @@
+import _ from 'lodash';
+
+const isRootLocaleComment =
+  reply => (reply.depth === 1 && reply.body.length === 4 && reply.body.substring(0, 2) === '# ');
+
+export const toObject = (body) => {
+  const object = {};
+  const lines = body.split(/\n/);
+  lines.forEach((line) => {
+    const message = line.split(':');
+    if (message.length === 2) {
+      object[message[0].trim()] = message[1].trim();
+    }
+  });
+  return object;
+};
+
+export const getMessages = (replies) => {
```

Comment:
I have security concerns over this function. There is no checks for the username or number of votes so the latest comment will override the translations in that locale. For example if I add a comment after you with the root comment of `FR` and write some french curses inside it it will be the primary language of the platform for french language which is an obvious vulnerability to attacks.

The first way to prevent it is to pick the root comment with the most upvotes between similar other locales but even doing so doesn't guarantee people using bots to promote a no-acceptable translation into the system. For now I suggest using a whitelist of users so if the username is not in that whitelist we just ignore it. It's easy to update the whitelist (could be a simple array in config.json) when we see a community voted translation is introduced.